core/src/main/java/com/bankbazaar/tripmanager/model/TripUserMapping.java

```diff
@@ -44,4 +45,12 @@ public class TripUserMapping implements Serializable {
     @LastModifiedDate
     @Column(name = "last_modified_date")
     private Date lastModifiedTime;
+
+    public TripUserMapping(Long tripId, Long userId, UserRole role) {
```

Comment:
Add an @AllArgsCountructor. Dont need to define explicitly

Reply:
This has been resolved